RInAppMessaging/Classes/Views/FullView.swift

```diff
@@ -109,6 +110,17 @@ internal class FullView: UIView, FullViewType, RichContentBrowsable {
         bodyViewOffsetYConstraint.constant = hasImage ? 0 : uiConstants.bodyViewSafeAreaOffsetY
     }
 
+    override func hitTest(_ point: CGPoint, with event: UIEvent?) -> UIView? {
+        let exitButtonCenter = convert(exitButton.center, from: exitButton.superview)
+        let exitButtonTouchArea = CGRect(origin: exitButtonCenter, size: .zero)
+            .insetBy(dx: -uiConstants.exitButtonTouchAreaSize / 2.0, dy: -uiConstants.exitButtonTouchAreaSize / 2.0)
+
+        if exitButtonTouchArea.contains(point) {
+            return exitButton
```

Comment:
I don't really like that there's near-identical duplicated code in our view classes.
Can it go into a common subclass of UIView?

Reply:
Yeah I don't like that either. I probably can move most of that code to some common place

Reply:
Do you want to do it in this PR or a follow-up?

Reply:
@donnie-jp This PR. I'll updade in a moment

Reply:
@donnie-jp Is this ok enough now?
I could move whole method to `BaseView` but then I'd have to bring the concept of exit button as well (outlets, constants etc). The other option would be to have some generic `hitTestViews` collection in BaseView that will be used in hitTest() to ensure min touch size (44pt constant defined in BaseView)

Reply:
Extension seems fine. Thanks for updating.